.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**I am currently running**
+<!-- If you don't use the latest version, please tell us why. -->
+- SpongeVanilla version: 
+- Java version: 
+- Operating System:
+
+<!-- Please include ALL mods/plugins you had installed when your issue happened, you can get a list of 
+        your mods and plugins by running "/sponge plugins" and/or "/sponge mods" -->
+- Plugins/Mods:
+
+<!-- Please include as much information as possible. For the description, assume we have no idea how 
+        mods work, be as detailed as possible and include a step by step reproduction. It is recommended 
```

Comment:
mods > things
or something like that, mods dont really apply here

Reply:
mods -> your setup?